docs/adding_a_benchmark.md

```diff
@@ -0,0 +1,40 @@
+## Adding a new Benchmark 
+
+MTEB covers a wide variety benchmark which are all presented in the public [leaderboard](https://huggingface.co/spaces/mteb/leaderboard). However many languages or domains are still missing benchmark and we welcome contributions.
+
+To add a new benchmark you will need to:
+
+1) [Implement the tasks](adding_a_dataset.md) that you want to evaluate on, or find then in the existing list of tasks.
+2) Implement the benchmark in the [`benchmark.py`](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/benchmarks/benchmarks.py) file and submit it as a PR
+
+This is easy to do 
+```python
+tasks = mteb.get_tasks(tasks=[] ...) # fetch the tasks you want to include in your benchmark
+
+MY_BENCHMARK = Benchmark(
+    name="Name of your benchmark",
+    tasks=tasks,
+    description="This benchmark tests y, which is important because of X",
+    reference="https://relevant_link_eg_to_paper.com",
+    citation="A bibtex citation if relevant",
+)
+```
+
+3) Run a representative set of models on benchmark and submit the results you do this by
+<!-- TODO: we should probably create seperate page for how to submit results -->
+1. Open a PR in https://hf.co/datasets/mteb/results with:
```

Comment:
You want to move results repository to HF?

Reply:
def. not - I will fix this